front/app/components/IdeasMap/utils.tsx

```diff
@@ -0,0 +1,84 @@
+import { colors, media } from '@citizenlab/cl2-component-library';
+import { maxPageWidth } from 'containers/ProjectsShowPage/styles';
+import styled from 'styled-components';
+
+// BELOW: Custom handling for idea map width
+// Description: This was existing styling prior to Esri migration.
+// TODO: Cleanup these styles
```

Comment:
As stated here - this code is from the previous idea map implementation. I don't have time to redo the styling at the moment, so I've added a TODO here to clean this up at some point.